blackbox/docs/sql/reference/create_table.txt

```diff
@@ -265,6 +265,18 @@ to 1000 milliseconds.
 
 For further details see :ref:`refresh_data` or :ref:`sql_ref_refresh`.
 
+.. _sql_ref_write_wait_for_active_shards:
+
+``write.wait_for_active_shards``
+................................
+
+  Specifies the number of shard copies that need to be active for the write
+  operation to proceed. If less shard are active the operation will timeout.
```

Comment:
Maybe extend this to "If less shards are active the operation will wait for 30s for them to become active or timeout"